meta-grpc-client/src/test/java/net/opentsdb/aura/metrics/meta/grpc/TestUtil.java

```diff
@@ -0,0 +1,52 @@
+package net.opentsdb.aura.metrics.meta.grpc;
+
+import org.roaringbitmap.RoaringBitmap;
+
+import java.io.ByteArrayOutputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.Iterator;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TestUtil {
+    public static int[] getTimestampSequence(int count) {
+        final int epochSecond = (int) Instant.now().getEpochSecond();
+        final int[] array_count = new int[count];
+        array_count[0] = epochSecond - epochSecond % 3600 * 6;
+
+        for(int i = 1 ; i < count; i++) {
+            array_count[i] = array_count[i-1] - 3600 * 6;
+        }
+
+        return array_count;
+    }
+
+    public static void verifyBitMap(RoaringBitmap bitMap, int[] ts) {
+        final Iterator<Integer> iterator = bitMap.iterator();
+        int i = 0;
+        for (; i < ts.length; i++) {
+            assertTrue(bitMap.contains(ts[i]));
```

Comment:
Is the bitmap not ordered? Do we not care about the order?

Reply:
Yeah seems like it is not ordered. We dont care about the order.